client/src/components/Button/Button.css

```diff
@@ -15,47 +15,47 @@
 .button:hover {
   background-color: #fcf0ee;
 }
-.button-large {
+.button--large {
   padding: 0.13em 2.71em;
   font-size: 36px;
 }
-.button-small {
+.button--small {
   padding: 0.42em 3.35em;
   font-size: 18px;
 }
-.button-pink {
+.button__pink {
```

Comment:
This was my fault. It was not clear in the presentation. 

A pink variant of the component would be .Button--pink
I think you only have variants here so all of these should be -- not __
In my example, I had a subsection of my component that was purple. So I had .Component__purple. That was confusing. sorry!